### DISCUSSION TITLE
Support multiple LIBRARY_ROOT directories

### DISCUSSION THREAD

**Discussion by MarcSeebold** [zoriya/Kyoo#438] [General]
### Feature description

I have two drives I'd like to use as my library:
- /ssd/media
- /ssd4tb/media

I tried creating a new folder with two softlinks, but it seems that Kyoo doesn't follow softlinks.

Two possible solutions:
1. Add support for multiple library roots
2. Follow softlinks

**Comment by zoriya** [comment]
Library are mounted as docker volumes, symlinks across volumes mostly don't work in docker.
You can effectively use multiples directory by manually editing the docker-compose.yml file. You'll need to change

```yml
  volumes:
    - ${LIBRARY_ROOT}:/video:ro
```

by

```yml
  volumes:
    - /ssd/media:/video/ssd:ro
    - /ssd4tb/media:/video/ssd4tb:ro
```

If you do change that, you'll need this edit in two places: in the x-transcoder's volume definition and in the scanner's volume definition.

If this is something that people often needs, we might want to look into easier ways of doing that ; for example using reusable volumes scopes for the transcoder/scanner.

(i'm closing this, but feel free to comment if you need more help!)

**Comment by vrozaksen** [comment]
Hi! I'm on the same boat with:
```
/mnt/user/data/media
/mnt/user/zfs_data/media
```
Also it would be nice to be more specific what folders we want here. Let's say:
```
/mnt/user/data/media/movies
/mnt/user/data/media/tv
/mnt/user/zfs_data/media/anime
/mnt/user/zfs_data/media/animemovies
```
